packages/core/src/App/AppContent.tsx

```diff
@@ -49,7 +53,18 @@ const AppContent: React.FC<{ passthrough: unknown }> = observer(({ passthrough }
                 is_passkeys_supported && isServicePasskeysFFEnabled && isWebPasskeysFFEnabled && isMobile
             );
         }
-    }, [isServicePasskeysFFEnabled, isGBLoaded, isWebPasskeysFFEnabled, is_passkeys_supported, isMobile, store.client]);
+        if (isPhoneNumberVerificationEnabled) {
+            store.client.setIsPhoneNumberVerificationEnabled(isPhoneNumberVerificationEnabled);
+        }
+    }, [
+        isServicePasskeysFFEnabled,
+        isGBLoaded,
+        isWebPasskeysFFEnabled,
+        is_passkeys_supported,
+        isMobile,
+        store.client,
+        isPhoneNumberVerificationEnabled,
+    ]);
```

Comment:
since passkeys and phone number verification features are independent, isnt it better we handle them in 2 useeffects?

Reply:
sure let me change it 🙏